jenkins/automation/kubernetes/nightly-build-and-deploy.groovy

```diff
@@ -133,6 +139,24 @@ pipeline {
                 }
             }
         }
+
+        stage ("Changesetlog generation") {
+            when {
+                expression { "${env.cortxCluster_status}" == "SUCCESS" && "${env.qaSanity_status}" == "SUCCESS"  }
+            }
+            steps {
+                script {
+                    def changelog = build job: '/Release_Engineering/re-workspace/ap_space/changelog-generation', wait: true, propagate: false,
```

Comment:
Please test with http://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/Cortx-Automation/job/changelog-generation/

Reply:
Tested job - http://eos-jenkins.colo.seagate.com/job/Release_Engineering/job/re-workspace/job/nikhildev/job/Nightly-K8s-Build-and-Deploy-test/69/console